base-permissions/src/androidLibMain/kotlin/PermissionContext.kt

```diff
@@ -14,8 +14,12 @@
     limitations under the License.
 
  */
-package com.splendo.kaluga.permissions.camera
 
-import com.splendo.kaluga.permissions.PermissionsBuilder
+package com.splendo.kaluga.permissions
 
-internal actual fun PermissionsBuilder.registerCameraPermissionBuilder(context: Any?) = register(builder = CameraPermissionManagerBuilder(), permission = CameraPermission::class)
\ No newline at end of file
+import android.content.Context
+import com.splendo.kaluga.base.ApplicationHolder
+
+actual data class PermissionContext(val context: Context)
```

Comment:
why no typealias?

Reply:
Context is an abstract class so it wont allow that

Reply:
I assume because `Context` is an abstract class, but `NSBundle` is not. So we have to declare either `expect class` or 'expect abstract class'.

Reply:
TIL